src/components/editor/ChangeEditor.js

```diff
@@ -82,5 +123,37 @@ export default function Editor({ change, blocksContent, initialMarkdown, setMark
     })();
   });
 
-	return <BlockNoteView editor={editor} />;
-}
+  // Function to get document items for the suggestion menu
+  const getDocumentMenuItems = async (query) => {
+    const documents = await getProjectDocuments(projectId, getAuthToken());
+
+    return documents
+      .filter((doc) => doc.name.toLowerCase().includes(query.toLowerCase())
+          && doc.name !== documentName)
```

Comment:
Technically we can have 2 documents with the same name, it's an edge case but an id is unique so it's better to filter by ensuring `doc.id !== documentId` here